build.sbt

```diff
@@ -33,22 +33,20 @@ libraryDependencies += "org.slf4j" % "slf4j-api" % slf4jVersion
 libraryDependencies += "io.lemonlabs" %% "scala-uri" % scalaUriVersion
 
 //Testing
-libraryDependencies += "org.scalatest" %% "scalatest" % scalaTestVersion % "test"
-libraryDependencies += "org.scalacheck" %% "scalacheck" % scalacheckVersion % "test"
-libraryDependencies += "org.scalamock" %% "scalamock-scalatest-support" % scalaMockVersion % "test"
-libraryDependencies += "ch.qos.logback" % "logback-classic" % logbackVersion % "test"
+libraryDependencies += "org.scalatest" %% "scalatest" % scalaTestVersion % Test
+libraryDependencies += "org.scalacheck" %% "scalacheck" % scalacheckVersion % Test
+libraryDependencies += "org.scalamock" %% "scalamock" % scalaMockVersion % Test
+libraryDependencies += "ch.qos.logback" % "logback-classic" % logbackVersion % Test
 
 scalacOptions ++= Seq(
   "-deprecation",
-  "-Xfatal-warnings",
+  // "-Xfatal-warnings",
   "-feature", //Emit warning and location for usages of features that should be imported explicitly.
   "-encoding",
   "UTF-8",
   "-unchecked", //Enable additional warnings where generated code depends on assumptions
-  "-Yno-adapted-args",
```

Comment:
Why did we delete this one?

Reply:
Removed in [Scala 2.13](https://github.com/scala/scala/releases/tag/v2.13.0):

![Screenshot 2020-01-24 at 11 56 19](https://user-images.githubusercontent.com/3115191/73064026-8da03e00-3ea0-11ea-86bf-4c619f5f739d.png)